runtime/near-vm-runner/src/wasmer_runner.rs

```diff
@@ -259,6 +259,59 @@ pub fn run_wasmer<'a>(
     }
 }
 
+pub(crate) fn run_wasmer_module<'a>(
+    module: Module,
+    method_name: &str,
+    ext: &mut dyn External,
+    context: VMContext,
+    wasm_config: &'a VMConfig,
+    fees_config: &'a RuntimeFeesConfig,
+    promise_results: &'a [PromiseResult],
+    profile: ProfileData,
+    current_protocol_version: ProtocolVersion,
+) -> (Option<VMOutcome>, Option<VMError>) {
+    if method_name.is_empty() {
+        return (
+            None,
+            Some(VMError::FunctionCallError(FunctionCallError::MethodResolveError(
+                MethodResolveError::MethodEmptyName,
+            ))),
+        );
+    }
```

Comment:
That's interesting! In principle, we could move this error to prepare stage. But that would be a bad idea, because we need to preserve semantics of sequential execution. More generally, prepare call should never fail, all errors should be moved to the moment we actually run the call.

Reply:
Also current semantics of preloader is that we preload the contract, and select method to execute later on.